pkg/gen/node-templates/package.json.mustache

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@pulumi/kubernetes",
-    "version": "{{ProviderVersion}}",
+    "version": "${VERSION}",
```

Comment:
This file needn't be a template anymore, so if you would prefer I could just check it in as is to `pack/nodejs/package.json`.  Whatever is your preference.